src/index.js

```diff
@@ -18,12 +18,23 @@ const pkgJson = require('../package.json');
 module.exports = {
   ...require('./big-date'),
   ...require('./recording'),
-  ...require('./log'),
   ...require('./serialize-json'),
   ...require('./coralogix'),
   ...require('./bunyan'),
   ...require('./winston'),
   ...require('./secret'),
+  ...require('./messageFormats'),
+  ...require('./utils'),
+  ...require('./rootLogger'),
+  ...require('./globalLogger'),
+  InterfaceBase: require('./InterfaceBase'),
```

Comment:
Please adjust these to use the same import style…

Reply:
> Please adjust these to use the same import style…

yes.